compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -181,6 +186,23 @@ function finishCallStmt(Tokenizer tok, CallStmt expr) returns Stmt|err:Syntax {
     return stmt;
 }
 
+function finishCheckingCallStmt(Tokenizer tok, CheckingKeyword checkingKeyword) returns CallStmt|err:Syntax {
+    Token? t = tok.current();
+    if t is "check"|"checkpanic" {
+        check tok.advance();
+        return { checkingKeyword, operand: check finishCheckingCallStmt(tok, t) };
+    }
+    else if t == "(" {
+        return { checkingKeyword, operand: check parseMethodCallStmt(tok) };
+    }
+    Expr operand = check parsePrimaryExpr(tok);
+    if operand is FunctionCallExpr|MethodCallExpr {
+        check tok.expect(";");
+        return { checkingKeyword, operand };
+    }
+    return parseError(tok, "function call, method call or checking expression expected");
```

Comment:
If we are being precise, this has to be `s/checking expression/checking statment`, but I though it would be confusing.

Reply:
Even `"function call or method call expected"` might be enough, since it's unlikely one would want a checking statement with checking expression in it.